### PR TITLE
Doc check-missing-processing

### LINKED ISSUE #222: Missing documentation on check-missing-processing iotool
`clinica iotool check-missing-processing` is not described in the documentation.

### PR DESCRIPTION
**Pull request by 14thibea**
Closes #222 .

### CODE REVIEW

File: clinica/iotools/converter_utils.py [hunk 1]
```diff
@@ -96,7 +96,10 @@ def print_longitudinal_analysis(
             for subject in subjects_avail:
                 subj_tsv_path = path.join(bids_dir, subject, f"{subject}_sessions.tsv")
                 subj_df = pd.read_csv(subj_tsv_path, sep="\t").set_index("session_id")
-                if ses in subj_df.index.values:
+                if (
+                    ses in subj_df.index.values
+                    and "diagnosis" in subj_df.columns.values
+                ):
```
Comment (ghisvail) [inline clinica/iotools/converter_utils.py:102]: That has nothing to do with this PR right?

**Review by ghisvail — COMMENTED** [review]
LGTM, with a comment on the `converter_utils` diff

File: clinica/iotools/converter_utils.py (hunk 1 above)
Comment (ghisvail) [reply clinica/iotools/converter_utils.py:102]: Looks like your first commit should have been in a separate PR maybe?

File: docs/IO.md [hunk 2]
```diff
@@ -43,7 +43,10 @@ Starting from a BIDS compliant dataset, this command creates:
 1. A TSV file for each session available with the list of the modalities found for each subject.
 The name of the files produced will be `<prefix>_ses-<session_label>.tsv`.
 2. A text file containing the number and the percentage of modalities missing for each session.
-The name of the files produced will be `<prefix>_summary.txt`.
+The name of the file produced will be `<prefix>_summary.txt`.
```
Comment (omar-rifai) [inline docs/IO.md:46]: nitpicking but maybe start directly with the name of the file (e.g <prefix>summary.txt: A text file containing the [...])

Comment (14thibea) [reply docs/IO.md]: Done in https://github.com/aramis-lab/clinica/pull/272/commits/6956e994ee02fb6da032f0c011243cd905b4dd3f

File: clinica/iotools/converter_utils.py (hunk 1 above)
Comment (14thibea) [reply clinica/iotools/converter_utils.py:102]: I realized this problem existed when I wrote the documentation, this is why I added it here.

**Pull request merged**